packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -215,3 +215,10 @@ export function computeSize(entry: RumPerformanceResourceTiming) {
 export function isAllowedRequestUrl(configuration: RumConfiguration, url: string) {
   return url && !configuration.isIntakeUrl(url)
 }
+
+const DATA_URL_REGEX = /data:([-\w]+\/[-+\w.]+)?(;?\w+=[-\w]+)*(;base64)?/g
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Looking at the [spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs) the format of data urls look like this: `data:[<mediatype>][;base64],<data>`.
So the regex could be simplified as follow: `data:(.+)(;base64)?,`

Reply:
This has been simplified